_source/_posts/2021-07-08-jenkins-ci-dotnet-update-secrets.md

```diff
@@ -0,0 +1,403 @@
+---
+layout: blog_post
+title: How to Update App Secrets with Jenkins CI and .NET Core
+author: nick-fisher
+by: contractor
+communities: [.net]
+description: "Learn how to use Jenkins continuous integration platform for managing the applciation secrets and credentials in .NET Core."
+tags: [csharp-8, csharp-9, dotnet, dotnetcore, csharp, aspnet, aspnetcore, dotnet5, jenkins, ci, continous-integration, credentials]
+tweets:
+- "Learn how to use #Jenkins + #Okta + #dotnet together with #continousintegration for your application's credentials and secrets ->"
+- "Get up to speed on using #Jenkins and #Okta for #dotnet ci/cd in this post"
+- "Wondering how to manage your application's secrets and credentials? We've got you covered with this post using #Jenkins, #dotnet and #Okta!"
+image: blog/featured/okta-dotnet-books-quarter.jpg
+type: conversion
+---
+
+## Introduction
+
+Jenkins is a free and open-source application that makes it easy to create CI/CD pipelines in almost any language or environment. Jenkins features a vast number of plugs to help create a CI/CD environment that is tailored to your technologies.  
+
+In this application, you will create a .NET5 MVC web application and check it into a git repository. You will secure this application with Okta. Finally, you will set up a Jenkins project to fetch the code from your git repository, publish the project, apply configuration values, and run the application. Combining these three technologies you'll see how quick and easy it is to set up a modern CI/CD environment for a web application that is properly secured with a great single sign-on provider.
+
+### What you'll need
+
+> Your favorite IDE. I will be using Visual Studio but Visual Studio Code is another great option
+> A git repository
+> .NET 5
+> Jenkins
+> A Web Browser to Access Jenkins
+> An Okta Developer Account
+> The Okta CLI tool
```

Comment:
This section didn't render correctly, though I have no idea why
Maybe replace the `>` with `-` (or `*`)